v0/sources/refiner/trackMapping.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "sourceKeys": "triggered_event",
+    "destKeys": "event"
+  },
+  {
+    "sourceKeys": "remote_id",
+    "destKeys": "userId"
+  },
+  {
+    "sourceKeys": "response.received_at",
+    "destKeys": "originalTimestamp"
+  },
+  {
+    "sourceKeys": "response",
+    "destKeys": "properties.response"
+  },
+  {
+    "sourceKeys": "form.uuid",
+    "destKeys": ["properties.refiner_form_uuid", "context.formUuid"]
```

Comment:
we are mapping context.formUuid in identify mapping as well. These two forms are same? if not, the values will be overlapped